core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -1258,10 +1265,15 @@ object AutoTuner extends Logging {
   ): AutoTuner = {
     try {
       val clusterPropsOpt = loadClusterProps(workerInfoFilePath)
-      new AutoTuner(clusterPropsOpt.getOrElse(new ClusterProperties()), singleAppProvider, platform,
-        driverInfoProvider)
+      val autoT = new AutoTuner(clusterPropsOpt.getOrElse(new ClusterProperties()),
+        singleAppProvider, platform, driverInfoProvider)
+      if(clusterPropsOpt.isEmpty) {
+        autoT.appendComment(s"workerInfo file not found at $workerInfoFilePath. " +
```

Comment:
Add a comment here to explain why we are making this check.
This makes the code more readable

```
// In case the workerInfo input path was incorrect, the AutoTuner needs to add a comment that the recommendations were generated by assuming default worker properties.
```

Reply:
Thanks @amahussein . This has been updated.